examples/repo_example/succinct_hash_bin_delegations.py

```diff
@@ -0,0 +1,233 @@
+# Copyright New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+"""
+A TUF succinct hash bin delegation example using the low-level TUF Metadata API.
+
+The example code in this file demonstrates how to *manually* perform succinct
+hash bin delegation using the low-level Metadata API.
+Succinct hash bin delegation achieves a similar result as using a standard hash
+bin delegation, but it requires less memory overhead.
```

Comment:
less memory overhead is likely true as well but I think the advantage we're looking for is smaller delegating json files: fewer bytes to transfer and and to parse.